pyaedt/rpc/rpyc_services.py

```diff
@@ -157,6 +157,10 @@ def makedirs(self, remotepath):
         self.client.root.makedirs(remotepath)
         return "Directory created!"
```

Comment:
```suggestion
        return "Directory created."
```
Avoid use of exclamation points in technical doc (and in general).